docs/guides/slp-software-setup.md

```diff
@@ -41,6 +41,7 @@ Before moving on with the scripting, you’ll need some SLP addresses of your ow
 
 First off, let’s get you an SLP compatible wallet. This will be the place where your freshly minted SLP tokens will reside, and also where you’ll have to send the required BCH to do so. After installing the software, we will make sure to keep the SLP address and it’s WIF.
 
+If you'd rather try things on testnet, you'll find a small comment after each section indicating the changes needed. To get testnet coins you can find several services, like the [bitcoin.com Testnet BCH faucet](https://developer.bitcoin.com/faucets/bch/).
```

Comment:
Would be good to add https://faucet.fullstack.cash/ as well since its more maintained